README.rst

```diff
@@ -32,24 +32,28 @@ Running Tests
 
 You can run all the tests by executing:
 
-    python -m venv venv
+    python -m venv myvirtualenvironmentname
     source venv/bin/activate
     python setup.py test
 
     # Alternatively you could run the test runner stand-alone
     pip install -r tests/requirements.txt
     python tests.settings.py
 
+    # Or on MacOS use Pyenv:
+    pyenv virtualenv myvirtualenvironmentname
+    pyenv activate myvirtualenvironmentname
```

Comment:
When using python3 pyenv is redundant.

Reply:
Agreed, the above '-m venv' is built into python which makes it platform independent and works for Mac, Linux and Windows

Reply:
Please remove